### PR TITLE
website: adding new features and layouts

### PR DESCRIPTION
**Pull request by asettle**
Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

**Comment by asettle** [comment]
Screenshot from 2021-04-21 13-26-59

File: _config.yml [hunk 1]
```diff
@@ -1,11 +1,25 @@
 title: Aquarium by Aquarist Labs
 
-description: >-
-  Experimenting on captive cephalopods since 2021
+description: >
+  This work is licensed under a Creative Commons
```
Comment (l-mb) [inline _config.yml:4]: Why this change? This is the short summary of the site, tagline etc; is that the best place for the license statement?

Comment (asettle) [reply _config.yml:4]: Ah, because I changed the footer. I'll show you the output

Comment (asettle) [reply _config.yml:4]: Screenshot from 2021-04-21 14-11-30

Now it's more inline with other website styling.

**Review by l-mb — APPROVED** [review]
LGTM to me, we can finetune later!
